### PR TITLE
Add package tag

### PR DESCRIPTION
**Pull request by emiura**
This "Package:" adds list of packages used on the testing

Fixes poo#77002

### CODE REVIEW

**Review by okurz — APPROVED** [review]
Well, there is nothing added to ensure the validity of the package tag but it's ok as it is

**Review by tjyrinki — APPROVED** [review]
Looks good and I don't see any errors. This would be a step forward in automating package coverage statistics.

File: tests/x11/piglit/piglit.pm [hunk 1]
```diff
@@ -7,6 +7,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package; xterm piglit
```
Comment (ggkioulis) [inline tests/x11/piglit/piglit.pm:10]:
```suggestion
# Package: xterm piglit
```

Comment (pdostal) [reply tests/x11/piglit/piglit.pm:10]: That's sight!

**Pull request merged**